docs/release-notes/2.26.0.md

```diff
@@ -0,0 +1,89 @@
+ 
+!!! caution alert alert-warning "Disclaimer"
+    Integrated Alerting is still a Preview functionality and, as such, subject to change. We recommend that early adopters use this feature for testing purposes only.
+
+
+## New Features
+
+* [PMM-9059](https://jira.percona.com/browse/PMM-9059): Following the recently introduced support for connecting PMM to Percona Platform, you now have the option to also unlink any servers that are no longer relevant to your Platform organization.
+ To disconnect a PMM server, go to **Configuration > Settings > Percona Platform** and click **Disconnect**.
+ You can check the list of servers connected to an organization in Percona Platform by clicking View instances on the Dashboard page. For more information, see the [Configure](https://www.percona.com/doc/percona-monitoring-and-management/2.x/how-to/configure.html) for more details.
+* [PMM-9312](https://jira.percona.com/browse/PMM-9312): Tech Preview Feature: PMM now captures the MongoDB metrics such as dbStats, collStats, indexStats, and topmetrics. See [Documentation](https://www.percona.com/doc/percona-monitoring-and-management/2.x/details/commands/pmm-admin.html) for more details.
+
+
+
+## Improvements
+
+* [PMM-9176](https://jira.percona.com/browse/PMM-9176): DBaaS: PMM now supports Percona Distribution for MongoDB Operator 1.10.0.
+* [PMM-9159](https://jira.percona.com/browse/PMM-9159): DBaaS: PMM now supports Percona Kubernetes Operator for Percona XtraDB Cluster 1.10.0.
+* [PMM-9180](https://jira.percona.com/browse/PMM-9180): Integrated Alerting:  Add Alert Rule: Added the Template expression in a collapsible panel for an enhanced user experience (default view as collapsed) as the technical message could confuse the users.
+* [PMM-7781](https://jira.percona.com/browse/PMM-7781): Integrated Alerting: Alert rules no longer depend on their source rule template after creation. This means that you can now update or delete rule templates without impacting existing rules that are based on that template. For more information, see [Integrated Alerting](../using/alerting.md).
+* [PMM-9356](https://jira.percona.com/browse/PMM-9356): Added new experimental Environment dashboards in PMM.
+* [PMM-9296](https://jira.percona.com/browse/PMM-9296): Disclaimer about Technical Preview feature added to Percona Platform - Connect PMM to Percona portal page.
+
+
+## Bugs Fixed
+
+* [PMM-8867](https://jira.percona.com/browse/PMM-8867): Fixed an issue for PMM Client installation using the tarball script (without using RPMs) where the configuration was getting lost due to the configuration file (pmm-agent.yml) being recreated.
```

Comment:
We should avoid content in parentheses as much as possible. We could move the info to a separate sentence instead?